client/web/src/App.jsx

```diff
@@ -168,12 +189,27 @@ const App = () => {
   }
 
   // Use custom hooks
-  const { send, connectSocket, closeSocket } = useWebsocket(handleSocketOnOpen,handleSocketOnMessage);
+  const { socketRef, send, connectSocket, closeSocket } = useWebsocket(token, handleSocketOnOpen,handleSocketOnMessage, selectedModel);
```

Comment:
it seems like, if the user is already login when they open the page, the token will be empty.